libcxx/test/std/cstring/freestanding/strtok_removed.fail.cpp

```diff
@@ -0,0 +1,14 @@
+// UNSUPPORTED: libcpp-has-no-cstring
+// REQUIRES: freestanding
+// XFAIL: *
+
+// RUN: %clang %s -c -o /dev/null -ffreestanding --std=c++2c 2>&1 | FileCheck %s
+
+#include <cstring> 
+
+int main() {
```

Comment:
I wonder whether this really works as `main` is only expected to be the entry in hosted environments.

Moreover, I think it's worthwhile to test all other freestanding functions in `<cstring>`/`<string.h>` together. Feature-test macro `__cpp_lib_freestanding_cstring` covers them.